src/core.js

```diff
@@ -739,26 +742,29 @@ $.extend( $.validator, {
 			return undefined;
 		},
 
-		defaultMessage: function( element, method ) {
-			return this.findDefined(
-				this.customMessage( element.name, method ),
-				this.customDataMessage( element, method ),
-
-				// 'title' is never undefined, so handle empty string as undefined
-				!this.settings.ignoreTitle && element.title || undefined,
-				$.validator.messages[ method ],
-				"<strong>Warning: No message defined for " + element.name + "</strong>"
-			);
-		},
+		defaultMessage: function( element, rule ) {
```

Comment:
You're absolutely right!
But just a note, undocumented methods are, in the first place, used internally, and only a bunch o methods are intended to be used elsewhere:

>The validate method returns a Validator object that has a few public methods that you can use to trigger validation programmatically or change the contents of the form. The validator object has more methods, but only those documented here are intended for usage.
>
* Validator.form() – Validates the form.
* Validator.element() – Validates a single element.
* Validator.resetForm() – Resets the controlled form.
* Validator.showErrors() – Show the specified messages.
* Validator.numberOfInvalids() – Returns the number of invalid fields.
>
There are a few static methods on the validator object:
>
* jQuery.validator.addMethod() – Add a custom validation method.
* jQuery.validator.format() – Replaces {n} placeholders with arguments.
* jQuery.validator.setDefaults() – Modify default settings for validation.
* jQuery.validator.addClassRules() – Add a compound class method.
>
...

But I can fix that in a followup proposal to maintain backward compatibility with code that still relaying on the old behavior.

Thanks for the note!